Source/WebCore/dom/Document.cpp

```diff
@@ -10894,6 +10903,13 @@ void Document::setRenderingIsSuppressedForViewTransitionAfterUpdateRendering()
     m_enableRenderingIsSuppressedForViewTransitionAfterUpdateRendering = true;
 }
 
+void Document::setRenderingIsSuppressedForViewTransitionImmediately()
```

Comment:
Should we add a FIXME to remove this once we fix the timing of capturing the old state?